content/en/docs/tasks/inject-data-application/define-environment-variable-container.md

```diff
@@ -1,58 +1,60 @@
 ---
-title: Define Environment Variables for a Container
-content_template: templates/task
```

Comment:
As in the other file, please revert the frontmatter areas and shortcodes to use the Hugo syntax throughout this file.